core/src/main/kotlin/transformers/descriptors/DefaultDescriptorToDocumentationTranslator.kt

```diff
@@ -27,119 +30,170 @@ object DefaultDescriptorToDocumentationTranslator: DescriptorToDocumentationTran
         platformData: PlatformData,
         context: DokkaContext
     ) = DokkaDescriptorVisitor(platformData, context.platforms[platformData]?.facade!!).run {
-        packageFragments.map { visitPackageFragmentDescriptor(it, DRI.topLevel) }
+        packageFragments.map { visitPackageFragmentDescriptor(it, DRIWithPlatformInfo(DRI.topLevel, null, emptyList())) }
     }.let { Module(moduleName, it) }
 
 }
 
+
+data class DRIWithPlatformInfo(
+    val dri: DRI,
+    val expected: PlatformInfo?,
+    val actual: List<PlatformInfo>
+)
+
 class DokkaDescriptorVisitor(
```

Comment:
If this two classes can be made private or internal - do so. `DRIWithPlatformInfo` may be confusing and i don't think we want to make them part of the api.